src/modules/auth/api.js

```diff
@@ -10,31 +10,52 @@ export class SeaCatAuthApi {
 
 	module.exports = {
 		app: {
-			"OIDC_URL": 'http://localhost:3000/openidconnect',
+			BASE_URL: 'http://localhost:3000',
+			API_PATH: '/api',
+			SERVICES: {oidc: '/openidconnect', rbac: '/rbac'},
 			...
 	*/
 
-	constructor(config) {
-		
-		this.BaseURL = config.get('OIDC_URL');
+	constructor(config, props) {
+
+		this.BaseURL = config.get('BASE_URL');
+		this.ApiPath = config.get('API_PATH');
+		this.Services = config.get('SERVICES');
```

Comment:
You definitively don't need to work with `BASE_URL`, `API_PATH` and `SERVICES` ... just get axios using the service name.

Reply:
@ateska Well, I suppose it is needed at least as local variables in constructor. Because it forms the URL for login:
https://github.com/TeskaLabs/asab-webui/blob/implement/authorisation-on-tenants/src/modules/auth/api.js#L63
and services for axios https://github.com/TeskaLabs/asab-webui/blob/implement/authorisation-on-tenants/src/modules/auth/api.js#L43

Reply:
Then you rather need a new method on Application, with the similar/same logic as is in `createAxios()` - you don't want to spill URL logic all over the code base.

That new method `getApiURL(service)` should be also used by `createAxios()`